### ISSUE TITLE
Why two jQuery templates?

### ISSUE THREAD

**Issue by EvanCarroll** [umdjs/umd#69]
We have two templates

* jqueryPluginCommonjs.js
* jqueryPlugin.js

This is really stupid. We're making the learning curve complex because of one concern..

```javascript
} else if (typeof module === 'object' && module.exports) {
    // Node/CommonJS
    module.exports = factory(require('jquery'));
}
```

That's one branch in an else statement? Is that all there is too it? Who cares.. Browserify is huge and growing explosively. The time it takes to,

* See there is a difference
* Read which file you should use

isn't worth saving a branch on an if statement. Not to mention, we're making assumptions about how people are *using* UMD. The whole purpose of *UMD* is that it should *just work* if it at all possible.

tldr; ***DELETE jqueryPluginCommonjs.js, and move the conditional into jqueryPlugin.js***

**Comment by EvanCarroll** [comment]
This will also close out issue https://github.com/umdjs/umd/issues/23
